app/views/manage/mturk_auto/index.html.erb

```diff
@@ -19,24 +19,45 @@
       </small>
     </div>
   <% end %>
-<% end %>
+<% end %><br/>
 
-<%= form_tag mturk_auto_update_primary_jobs_path do %>
-  <%= fields_for :mturk_auto do |ff| %>
-    <h4 class="mb-3">Primary MTurk jobs</h4>
+<h4 class="mb-3">Primary MTurk jobs</h4>
+<%
+  text = '<p class="text-left"><small>Make sure that each project with auto MTurking turned on has a primary batch job to copy parameters from (except max tasks per worker and qualification list).</small><br/><br/>'
+  text += '<small class="text-danger">If a primary job is not set at the time of creation of a new auto batch, the creation will fail.</small><br/><br/>'
+  text += "<small>The <i>check_samples_status</i> rake task (which creates new auto MTurk batches) runs every hour, so in case you forget to set up a primary job, do it as soon as you remember.</small></p>"
+%>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Project name</th>
+      <th><span style="border-bottom: 1px dotted;" data-toggle="tooltip" data-placement="top" data-html="true" title="<%= text %>">
+        Primary job [question sequence]
+      </span></th>
+      <th>Max tasks per worker</th>
+      <th>Qualification list</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
     <% @primary_jobs.each do |primary_job| %>
-      <div class="form-group mb-3">
-        <div class="input-group">
-          <span class="input-group-text mr-1"><i><%= primary_job.project.name %></i></span>
+      <%= simple_form_for primary_job do |f| %>
+        <tr>
+          <td class="align-middle"><i><%= primary_job.project.name %></i></td>
           <% options = MturkBatchJob.includes(:project).where(auto: false, 'projects.name': primary_job.project.name) %>
           <% collection = options.collect{ |s| ["#{s.name} [#{s.project.question_sequence_name.nil? && s.project.primary? ? 'primary' : s.project.question_sequence_name}]", s.id] } %>
-          <%= ff.select "primary_jobs[#{primary_job.project.id}]", options_for_select(collection, primary_job.mturk_batch_job&.id), { include_blank: primary_job.mturk_batch_job&.id.nil? }, { class: "form-control" } %>
-        </div>
-      </div>
+          <td><%= f.select :mturk_batch_job_id, options_for_select(collection, primary_job.mturk_batch_job&.id), { include_blank: primary_job.mturk_batch_job&.id.nil? }, { class: "form-control" } %></td>
+          <td><%= f.text_field :max_tasks_per_worker, as: :integer, class: "form-control" %></td>
+          <% options_ql = MturkWorkerQualificationList.all %>
```

Comment:
This seems dangerously unscalable, but having an asynchronous drop down could perhaps be a separate issue? Or will this table always have fewer than 1000 records?

Reply:
This table will be at most the number of projects in the DB, which is currently 10 in production. But in reality, even less than that.

Reply:
> Thanks for the PR! 👍 It's my first time reviewing code on this project so please excuse any questions below that arise from unfamiliarity 🙂

It's the first time someone is reviewing my code, so please excuse any issues that arise from my blissful ignorance about Rails 😁